test/test.js

```diff
@@ -419,6 +419,30 @@ describe('conventionalChangelogCore', function() {
     }));
   });
 
+  it('should use custom referenceActions if a `referenceActions` is provided in the `options.config.parserOpts` object', function(done) {
+    gitDummyCommit('Custom prefix closes #42');
+    gitDummyCommit('Old prefix #71');
+
+    conventionalChangelogCore({
+      config: {
+        parserOpts: null
```

Comment:
Should this contain `referenceActions`, and `hostOpts.referenceActions` shouldn't overwrite the option?